cmd/link.go

```diff
@@ -68,6 +89,7 @@ If not, then please delete the service and recreate it using 'odo service create
 }
 
 func init() {
+	linkCmd.PersistentFlags().BoolVarP(&wait, "wait", "w", false, "If enabled, the link command will wait for the service to be provisioned if it's not yet")
```

Comment:
I feel like this can be shortened to: `If enabled, the link command will wait for the service to be provisioned`